storage/cache/log_subtree_cache.go

```diff
@@ -58,6 +58,23 @@ func populateLogSubtreeNodes(hasher hashers.LogHasher) storage.PopulateSubtreeFu
 		if st.InternalNodes == nil || len(st.Leaves) == maxLeaves {
 			st.InternalNodes = make(map[string][]byte)
 		}
+		store := func(id compact.NodeID, hash []byte) {
```

Comment:
Is there a clearer name than 'store'? Maybe not.

Reply:
Can't think of a better one :) Could be a longer one, but it seems pretty accurate if considered in context of its signature: "store a node with the specified ID and hash into a map".